paymentsheet/src/main/java/com/stripe/android/link/ui/inline/InlineSignupViewState.kt

```diff
@@ -83,13 +94,28 @@ constructor(
                 }
             }
 
+            val allowsDefaultOptIn = FeatureFlags.linkDoi.isEnabled &&
```

Comment:
can we maybe rename `linkDoi` to `linkDefaultOptIn`? Might be confusing for non link folks, especially in the playground.